citeproc/source/__init__.py

```diff
@@ -8,24 +8,39 @@
 from .. import VARIABLES
 
 
+class MissingArgumentWarning(UserWarning,TypeError):
+    pass
+
+class UnsupportedArgumentWarning(UserWarning,TypeError):
+    pass
```

Comment:
what is the point to subclass from TypeError if used only as a warning?

Reply:
I think my rationale was to make sure that any existing code that relied on "except TypeError: (...)" to catch this situation would continue working, assuming the warning is configured by default to raise an exception.

Reply:
(I agree it seems unnecessary unless you want to guarantee the same behavior in existing code, ie., raising a TypeError on missing arguments.)